### PR TITLE
Update dockerignore

### PR DESCRIPTION
**Pull request by marquiz**
Exclude more unneeded files/directories

### CODE REVIEW

**Comment by marquiz** [comment]
ping @balajismaniam, any comments

**Comment by marquiz** [comment]
ping @balajismaniam!

**Comment by marquiz** [comment]
@balajismaniam ping!

**Comment by marquiz** [comment]
updated

/remove-lifecycle stale

File: .dockerignore [hunk 1]
```diff
@@ -2,3 +2,5 @@ node-feature-discovery
 node-feature-discovery-job.json
 vendor/
 demo/
+.dockerignore
+.git*
```
Comment (Ethyling) [inline .dockerignore]: I think you can remove some files from it (like `node-feature-discovery`) and maybe sort it.

Comment (marquiz) [reply .dockerignore]: Thanks for looking at this. PR updated (sorted and the outdated entries removed/updated)

**Pull request merged**